src/options.rs

```diff
@@ -419,6 +422,7 @@ pub fn build_app(
         .current_widget(widget_map.get(&initial_widget_id).unwrap().clone()) // TODO: [UNWRAP] - many of the unwraps are fine (like this one) but do a once-over and/or switch to expect?
         .widget_map(widget_map)
         .used_widgets(used_widgets)
+        .is_expanded(expanded_upon_startup)
```

Comment:
Sorry for not catching this earlier, but we probably want to make this not fire if in basic mode (e.g. `btm --basic -e`), as expansion doesn't do anything in that mode at the moment.

Reply:
@ClementTsang  good shout. updated now